### PR TITLE
Add plugin crashmon

### PR DESCRIPTION
**Pull request by skvl**
This plugin monitors Windows process "WerFault.exe" which is called on process crash.

From command line of the process one could get PID of crashed process.

Also check for crashing processes while injection.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -443,4 +453,5 @@ Socketmon:  $plugin_socketmon
 Regmon:     $plugin_regmon
 Procmon:    $plugin_procmon
 BSODmon:    $plugin_bsodmon
+CrashMmon:  $plugin_crashmon
```
Comment (tklengyel) [inline configure.ac]: `Mmon`?

Comment (skvl) [reply configure.ac]: Sorry. Fixed!

File: src/plugins/crashmon/crashmon.cpp [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            char* tmp = drakvuf_get_process_name(drakvuf, eprocess, false);
+            if (tmp)
+                name = tmp;
+
+            if (VMI_FAILURE == drakvuf_get_process_ppid(drakvuf, eprocess, &ppid))
+                PRINT_DEBUG("[CRASHMON] Failed to get PPID for crached process with PID %d\n", pid);
+        }
+        else
+        {
+            PRINT_DEBUG("[CRASHMON] Failed to get EPROCESS for crached process with PID %d\n", pid);
```
Comment (tklengyel) [inline src/plugins/crashmon/crashmon.cpp]: `r/crached/crashed`

File: src/plugins/crashmon/crashmon.cpp [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        const char* name = "<UNKNOWN>";
+
+        if (drakvuf_find_process(drakvuf, pid, nullptr, &eprocess))
+        {
+            char* tmp = drakvuf_get_process_name(drakvuf, eprocess, false);
+            if (tmp)
+                name = tmp;
+
+            if (VMI_FAILURE == drakvuf_get_process_ppid(drakvuf, eprocess, &ppid))
+                PRINT_DEBUG("[CRASHMON] Failed to get PPID for crached process with PID %d\n", pid);
```
Comment (tklengyel) [inline src/plugins/crashmon/crashmon.cpp]: `r/crached/crashed`

Comment (skvl) [reply src/plugins/crashmon/crashmon.cpp]: Fixed!

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Pull request merged**